pkg/virt-handler/vm.go

```diff
@@ -641,6 +641,22 @@ func (d *VirtualMachineController) migrationOrphanedSourceNodeExecute(key string
 	return nil
 }
 
+func isMigrating(start, stop *v12.Time) bool {
```

Comment:
Perhaps let's change this to something like the following to make sure that `vmi.Status.MigrationState` isn't `nil`?

```suggestion
func isMigrating(vmi *v1.VirtualMachineInstance) bool {                         
                                                                                
    now := v12.Now()                                                            
                                                                                
    running := false                                                            
    if vmi.Status.MigrationState != nil {                                       
        start := vmi.Status.MigrationState.StartTimestamp                       
        stop := vmi.Status.MigrationState.EndTimestamp                          
        if start != nil && (now.After(start.Time) || now.Equal(start)) {        
            running = true                                                      
        }                                                                       
                                                                                
        if stop != nil && (now.After(stop.Time) || now.Equal(stop)) {           
            running = false                                                     
        }                                                                       
    }                                                                           
    return running                                                              
}               
```